modules/end-user-guide/pages/url-parameter-for-starting-ephemeral-workspaces.adoc

```diff
@@ -0,0 +1,23 @@
+:_content-type: CONCEPT
+:description: URL parameter starting ephemeral workspaces
+:keywords: ephemeral-workspace, persistent-workspace
+:navtitle: URL parameter starting ephemeral workspaces
+:page-aliases:
+
+[id="url-parameter-for-starting-ephemeral-workspaces_{context}"]
+= URL parameter for starting ephemeral workspaces
+
+If the URL for starting a new workspace doesn't contain a URL parameter specifying the storage type, {prod-short} will not persist by default the workspace and its data and they'll be lost when the workspace stops.
```

Comment:
```suggestion
If the URL for starting a new workspace doesn't contain a URL parameter specifying the storage type, {prod-short} does not persist by default. The workspace and its data are lost when the workspace stops.
```

Reply:
sure!

Reply:
@akurinnoy weird, this sentence looks ambiguous. AFAIK if the URL parameter is not part of the URL the default storage type will be used, no?